### PR TITLE
Chore/tri 1241 decentralized digital twins and registry

### PR DESCRIPTION
**Pull request by mkanal**
Add uml diagrams for decentral registry approach,

### CODE REVIEW

File: uml-diagrams/decentral-twin-registry/edc-service-discovery-cache.puml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+participant "IRS" as IRS
+participant "EDC Discovery Finder Cache" as CACHE
+participant "EDC Discovery Finder" as EDCDF
+
+    IRS -> CACHE : lookup endpoints for BPNL
+    alt cache contains endpoints for BPNL
+        IRS <-- CACHE: array of endpoints from cache
+    else
+         IRS -> EDCDF : POST /api/administration/connectory/discovery {"BPNL":"urn:..."}
```
Comment (ds-jhartmann) [inline uml-diagrams/decentral-twin-registry/edc-service-discovery-cache.puml:16]: Same as above. 
```suggestion
         IRS -> EDCDF : POST /api/administration/connectors/discovery [ "BPNL..." ]
```

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: irs-api/src/main/java/org/eclipse/tractusx/irs/services/BatchOrderEventListener.java
Comment (ds-jhartmann) [inline irs-api/src/main/java/org/eclipse/tractusx/irs/services/BatchOrderEventListener.java:1]: Why was this changed?

File: irs-api/src/test/java/org/eclipse/tractusx/irs/services/BatchOrderEventListenerTest.java
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/services/BatchOrderEventListenerTest.java:1]: Why was this changed?

File: irs-api/src/main/java/org/eclipse/tractusx/irs/services/BatchOrderEventListener.java
Comment (mkanal) [reply irs-api/src/main/java/org/eclipse/tractusx/irs/services/BatchOrderEventListener.java:1]: This was changed by KM.

File: uml-diagrams/decentral-twin-registry/decentral-digital-registry.puml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+             note left
+                 {
+                 	"id": "1701f7b6-e5fb-4cea-8196-391df58a8916",
+                 	"endpoint": "https://irs-consumer-dataplane.dev.demo.catena-x.net/api/public",
+                 	"authKey": "Authorization",
+                 	"authCode": "eyJoZUhBa....",
+                 	"properties": {
+                 		"cid": "dea8de11-b176-4d4b-a8c5-24c35cfd45ba:c54025a3-208e-479a-9370-a4cca3b3967d"
+                 	}
+                 }
```
[the remark this answers is not in the document]
Comment (mkanal) [reply uml-diagrams/decentral-twin-registry/decentral-digital-registry.puml:69]: Done
